packages/suite/src/views/dashboard/components/PortfolioCard/index.tsx

```diff
@@ -60,9 +64,22 @@ const PortfolioCard = React.memo(() => {
     const isWalletError = discoveryStatus?.status === 'exception' ?? false;
     const showGraphControls = !isWalletEmpty && !isWalletLoading && !isWalletError;
 
+    const showMissingDataTooltip =
+        showGraphControls &&
+        accounts.find(a => a.networkType === 'ethereum' || a.networkType === 'ripple');
```

Comment:
Would maybe cast this into a boolean, just to make sure we're working with the correct type.